gateleen-core/src/main/java/org/swisspush/gateleen/core/http/LocalHttpClientRequest.java

```diff
@@ -776,6 +797,8 @@ public boolean writeQueueFull() {
 
     @Override
     public HttpClientRequest drainHandler(Handler<Void> handler) {
+        log.warn("Happy debugging, as this impl will just ignore your drainHandler anyway",
```

Comment:
We expect or actually know that it we would log this newly added warning only in some rare edge cases, right?

Reply:
I have no idea when it is called. And that's exactly the point why I add this log. As soon someone calls this method (if at all), it is a guarantee for a hung-up request (eg timeout) with totally useless error messages where the stack points somewhere into vertx. The issue is, no one knows if this faulty method here is called at all due to indirections and abstractions.